server/noncommercial-schema.es6

```diff
@@ -0,0 +1,330 @@
+var noncommercialSchema = {
+  'id':'/noncommercialPermit',
+  'type':'object',
+  'allOf':[
+    { '$ref': '/commonFields'},
+    {
+      'properties':{
+        'applicantInfo': {
+          '$ref': '/noncommercialApplicantInfo'
+        },
+        'type': {
+          'default': 'noncommercial',
+          'enum':[
+            'noncommercial',
+            'tempOutfitters'
+          ],
+          'type': 'string'
+        },
+        'noncommercialFields': {
+          '$ref': '/noncommercialFields'
+        }
+      },
+      'required': ['applicantInfo', 'type', 'noncommercialFields']
+    }
+  ]
+};
+
+var noncommercialApplicantInfoSchema = {
+  'id': '/noncommercialApplicantInfo',
+  'type': 'object',
+  'allOf':[
+    {'$ref':'/applicantInfoBase'},
+    {
+      'properties': {
+        'organizationName': {
+          'default':'',
+          'type': 'string'
+        },
+        'orgType':{
+          'default':'',
+          'description':'Organization Type',
+          'enum':[
+            'Individual',
+            'Corporation',
+            'Limited Liability Company',
+            'Partnership or Association',
+            'State Government or Agency',
+            'Local Government or Agency',
+            'Nonprofit'
+          ],
+          'type':'string'
+        }
+      },
+      'dependencies':{
+        'organizationName':['orgType']
+      }
+    }
+  ]
+};
+
+var noncommercialFieldsSchema = {
+  'id': '/noncommercialFields',
+  'type': 'object',
+  'properties': {
+    'activityDescription': {
+      'default':'',
+      'type': 'string'
+    },
+    'locationDescription': {
+      'default':'',
+      'type': 'string'
+    },
+    'startDateTime': {
+      'default':'',
+      'pattern':'^(19|20)\\d\\d-(0[1-9]|1[012])-(0[1-9]|[12][0-9]|3[01])T(0\\d|1\\d|2[0-3]):(0\\d|1\\d|2[0-3]):(0\\d|1\\d|2[0-3])Z$',
```

Comment:
Any particular reason you used a string/regex instead of a dateTime object? (See the bottom of the page here: https://spacetelescope.github.io/understanding-json-schema/reference/string.html)